Source/ACE/Entity/Landblock.cs

```diff
@@ -467,6 +472,90 @@ private void HandleGameAction(QueuedGameAction action, Player player)
                         HandleSoundEvent(obj, soundEffect);
                         break;
                     }
+                case GameActionType.DropItem:                  
+
+                    {                        
+                        var g = new ObjectGuid(action.ObjectId);                        
+                        // ReSharper disable once InconsistentlySynchronizedField
+                        if (worldObjects.ContainsKey(g))
+                        {                         
+                            var playerId = new ObjectGuid(action.ObjectId);
+                            var inventoryId = new ObjectGuid(action.SecondaryObjectId);
+                            if (playerId.IsPlayer())
+                            {
+                                Player aPlayer = null;
+                                WorldObject inventoryItem = null;
+                                lock (objectCacheLocker)
+                                {
+                                    if (worldObjects.ContainsKey(playerId))
+                                    {
+                                        aPlayer = (Player)worldObjects[playerId];
+                                        inventoryItem = aPlayer.GetInventoryItem(inventoryId);
+                                        aPlayer.RemoveFromInventory(inventoryId);
+                                    }
+                                }
+                                
+                                // We are droping the item - let's keep track of change in burden
+

```

Comment:
Removing the item from the player's inventory should do all this, not the landblock.